examples/key/variables.tf

```diff
@@ -0,0 +1,46 @@
+#########################################################################################
```

Comment:
This a generic comment

Can we add README.md under examples directory to cover "How To" section from KMS doc